app/src/organisms/Devices/RobotSettings/RobotSettingsAdvanced.tsx

```diff
@@ -166,12 +166,13 @@ export function RobotSettingsAdvanced({
         <RobotServerVersion robotName={robotName} />
         <Divider marginY={SPACING.spacing16} />
         <RobotInformation robotName={robotName} />
-        <Divider marginY={SPACING.spacing16} />
+        isOt3 ? (<Divider marginY={SPACING.spacing16} />
         <UsageSettings
           settings={findSettings('enableDoorSafetySwitch')}
           robotName={robotName}
           isRobotBusy={isRobotBusy}
         />
+        ) : null
```

Comment:
I'm not in the office to test this but if we want this setting to not show for the Flex, your logic is backwards. Also i fixed a few syntax issues:

```suggestion
      {isOT3 ? null : (
          <>
            <Divider marginY={SPACING.spacing16} />
            <UsageSettings
              settings={findSettings('enableDoorSafetySwitch')}
              robotName={robotName}
              isRobotBusy={isRobotBusy}
            />
          </>
        )}
```